ui/components/app/collectible-details/collectible-details.js

```diff
@@ -85,6 +86,7 @@ export default function CollectibleDetails({ collectible }) {
 
   const onRemove = () => {
     dispatch(removeAndIgnoreNft(address, tokenId));
+    dispatch(setRemoveCollectibleMessage('success'));
```

Comment:
Can we collect the `success` message as an enum to use? `

Reply:
Yup, that's a great idea. Currently, I am following how we have done it for add NFT. So, I will include it in a future scope to update all the instances.